### PR TITLE
Implement block validator index page top part

### LINKED ISSUE #749: Implement block validator index page: top part
Header and top card.

### PR DESCRIPTION
**Pull request by prin-r**
Fix #749

### CODE REVIEW

File: scan/src/pages/ValidatorIndexPage.re [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    <VSpacing size=Spacing.xl />
+    <div className=Styles.vFlex>
+      <Text value="CoinGecko Data Provider" size=Text.Xxl weight=Text.Bold nowrap=true />
+    </div>
+    <VSpacing size=Spacing.xl />
+    <div className=Styles.topPartWrapper>
+      <Text value="INFORMATION" size=Text.Lg weight=Text.Semibold />
+      <VSpacing size=Spacing.lg />
+      {kvRow(
+         "OPERATOR ADDRESS",
```
Comment (taobun) [inline scan/src/pages/ValidatorIndexPage.re]: Test spacing seems wrong

File: scan/src/pages/ValidatorIndexPage.re [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+      <VSpacing size=Spacing.lg />
+      {kvRow("ADDRESS", VAddress("e38475F47166d30A6e4E2E2C37e4B75E88Aa8b5B" |> Address.fromHex))}
+      <VSpacing size=Spacing.lg />
+      {kvRow("VOTING POWER", VCode("45.34% (12,434.234 BAND)"))}
+      <VSpacing size=Spacing.lg />
+      {kvRow("COMMISSION", VCode("3.00%"))}
+      <VSpacing size=Spacing.lg />
+      {kvRow("BONDED HEIGHT", VCode("1"))}
+      <VSpacing size=Spacing.lg />
+      {kvRow("WEBSITE", VText("https://coingecko.node"))}
```
Comment (taobun) [inline scan/src/pages/ValidatorIndexPage.re]: Underline website?

File: scan/src/pages/ValidatorIndexPage.re [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+      <VSpacing size=Spacing.lg />
+      {kvRow(
+         "DETAILS",
+         VText("We are the leading staking service provider for blockchain projects."),
```
Comment (taobun) [inline scan/src/pages/ValidatorIndexPage.re:135]: Intentionally bold text?

**Review by taobun — CHANGES_REQUESTED** [review]

**Review by taobun — DISMISSED** [review]

File: scan/src/reusable/AddressRender.re [hunk 4]
```diff
@@ -36,18 +37,22 @@ module Styles = {
     fun
     | Title => style([pointerEvents(`none)])
     | Subtitle
-    | Text => style([pointerEvents(`auto)]);
+    | Text => style([pointerEvents(`auto)])
+    | None => style([pointerEvents(`none)]);
```
Comment (evilpeach) [inline scan/src/reusable/AddressRender.re]: Let's code like this.
```
| Title
| Subtitle
| None => style([pointerEvents(`none)])
| Text => style([pointerEvents(`auto)])
```

Comment (prin-r) [reply scan/src/reusable/AddressRender.re]: done

**Review by evilpeach — APPROVED** [review]

**Review by taobun — APPROVED** [review]

**Pull request merged**